pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We'd like this to ultimately be owned by graphql-java. Should this be a gradle project to match the other repos ([1](https://github.com/graphql-java/graphql-java-extended-scalars), [2](https://github.com/graphql-java/graphql-java), [3](https://github.com/graphql-java/java-dataloader)) in the org?

Reply:
That would be great, yes.

Reply:
Converted to gradle.